tests/unit/views/settings/Auth.spec.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable no-unused-expressions */
+import { expect } from 'chai'
+import sinon from 'sinon'
+import Auth from 'corteza-webapp-admin/src/views/Settings/Auth'
+import { stdReject, shallowMount } from 'corteza-webapp-admin/tests/lib/helpers'
+import fp from 'flush-promises'
+
+describe('views/Settings/Auth.vue', () => {
+  afterEach(() => {
+    sinon.restore()
+  })
+
+  let $SystemAPI
+  beforeEach(() => {
+    $SystemAPI = {
+      settingsUpdate: sinon.stub().resolves(),
+      settingsList: sinon.stub().resolves(),
+    }
+
+    sinon.stub(Auth, 'created')
+  })
+
+  const mountEP = (opt) => shallowMount(Auth, {
+    mocks: { $SystemAPI },
+    ...opt,
+  })
+
+  describe('update compose settings', () => {
+    it('on success', async () => {
+      const wrap = mountEP()
+      wrap.vm.onSubmit()
+
+      await fp()
+      sinon.assert.calledOnce($SystemAPI.settingsUpdate)
+      expect(wrap.vm.processing).to.be.false
+    })
+
+    it('on error - set error flag', async () => {
+      $SystemAPI.settingsUpdate = stdReject()
+      const wrap = mountEP()
+      wrap.vm.onSubmit()
+
+      await fp()
+      sinon.assert.calledOnce($SystemAPI.settingsUpdate)
+      expect(wrap.vm.error).to.not.be.null
+      expect(wrap.vm.processing).to.be.false
+    })
+  })
+
+  describe('fetch compose settings', () => {
+    it('on success - set internal compose settings', async () => {
+      const settings = [
+        { name: 'auth.external.enabled', value: false },
```

Comment:
You don't need that many cases for this since the method in question only maps { name, value } to { name: value }.

A few combinations with different data types should be enough.

Reply:
Same applies to all of these specs.